### PR TITLE
{chem}[intel/2018b] Bader v1.03

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/7d29ef4c01233cb3e5cd2babc257f71d for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/fe87af4deece9aaed1a99aa9288ffc37 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/0560ba6def13fb790c215e021820187a for a full test report.

File: easybuild/easyconfigs/b/Bader/Bader-1.03-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+homepage = 'http://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "A fast algorithm for doing Bader's analysis on a charge density grid."
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['http://theory.cm.utexas.edu/henkelman/code/bader/download/v%(version)s']
+sources = [{'download_filename': 'bader.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['3de8af94bd64c43812de7ebad3e9209cc5e29e6732679fcf88f72a1b382bc81a']
+
+skipsteps = ['configure']
```
Comment (akesandgren) [inline easybuild/easyconfigs/b/Bader/Bader-1.03-intel-2018b.eb]: There is no configure step to skip with MakeCp, you can however turn it on by using "with_configure = True"

Comment (boegel) [reply easybuild/easyconfigs/b/Bader/Bader-1.03-intel-2018b.eb]: +1, please drop

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ee52400307b6aa8a142168aa2d8aed23 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2098.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/816f5f759010be7ae134e4a0da71874a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3118.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/748378346616b46a4b743ab4beeb44ab for a full test report.

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**